README.md

```
@@ -1,4 +1,7 @@
-# Dart 수업 폴더 규칙
+# Dart 수업 폴더 규칙 #
+# Dart 수업 폴더 규칙 #
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)